src/components/NcAppContent/NcAppContent.vue

```diff
@@ -202,6 +207,17 @@ export default {
 			type: String,
 			default: null,
 		},
+
+		/**
+		 * Allow setting the page's `<title>`
+		 *
+		 * If a page heading is set it defaults to `{pageHeading} - {appName} - {productName}` e.g. `Favorites - Files - Nextcloud`.
+		 * When setting the prop then the following format will be used: `{pageTitle} - {pageHeading || appName} - {productName}`
```

Comment:
Can you add an example here for the second option? I cannot figure out what it can be...

Reply:
And docs for `pageHeading` also should be updated because now it not only setting `h1`, but also `<title>`

Reply:
Also, does it mean that a page now can never have a 2-part title like `Talk - Nextcloud`, `Mail - Nextcloud`, `Contacts - Nextcloud` and etc.?

And the only way to have such a title is to have neither heading nor title (and no `h1` which is a11y issue).

Reply:
@ShGKme  I pushed a commit with a fix for this.
For `Talk - Nextcloud` simply set `pageHeading` to `Talk`.